dspace-oai/pom.xml

```diff
@@ -87,18 +87,16 @@
             </exclusions>
         </dependency>
 
-        <!-- Templating Engine -->
+        <!-- Spring Boot Thymeleaf templating -->
         <dependency>
-            <groupId>org.jtwig</groupId>
-            <artifactId>jtwig-spring-boot-starter</artifactId>
-            <version>${jtwig.version}</version>
```

Comment:
Please also remove this `jtwig.version` property which is defined at the top of this POM file.

Reply:
thanks @tdonohue ! done. i also saw references in LICENSES_THIRD_PARTY and removed those